src/AutorestSwift/Code Generator/SwiftGenerator.swift

```diff
@@ -52,10 +52,20 @@ class SwiftGenerator: CodeGenerator {
     func generate() throws {
         let modelUrl = baseUrl.with(subfolder: .models, withTargetName: targetName)
         let optionsUrl = baseUrl.with(subfolder: .options, withTargetName: targetName)
+        let utilUrl = baseUrl.with(subfolder: .util, withTargetName: targetName)
         try modelUrl.ensureExists()
         try optionsUrl.ensureExists()
+        try utilUrl.ensureExists()
         logger.log("Base URL: \(baseUrl.path)")
 
+        // Create PatchUtil.swift file
```

Comment:
Should we check if the code model has patch operation before generating this PatchUtil.swift file?

Reply:
Shouldn't need to since it is going in the Util folder and not the Models folder. What we might need want to check is whether there is something in Models that shares the same name as something in PatchUtil... but since this is temporary anyways, we are probably okay.